src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -52,13 +52,12 @@
    * order to be relevance and max results to be 100, the API's arbitrary limit
    */
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
     try {
-      YouTube youtubeService = getService();
       String url = request.getParameter(URL_PARAMETER);
       CommentThreadListResponse commentResponse = generateYouTubeRequest(url).execute();
       // TODO: input commentResponse into commentAnalysis class to get useful data
-      Statistics statistics = new Statistics(new ArrayList<Double>(), 0);
+      Statistics statistics = new Statistics(new ArrayList<Double>());
       String json = new Gson().toJson(statistics);
       response.setContentType("application/json");
       response.getWriter().println(json);
```

Comment:
catch (IOException | ServletException e)  this is clearer

Reply:
Maybe IOException and ServletException are not enough.... Exceptions seem a little bit vague to me

Reply:
Haha I think Rick told me the opposite in one of his reviews actually. I think keeping it as just exception is good because it's vague actually, that way if any exception besides the two listed is encountered it can handle it as well.